src/components/Avatar.js

```diff
@@ -45,13 +57,24 @@ class Avatar extends PureComponent {
         ];
 
         const iconSize = StyleUtils.getAvatarSize(this.props.size);
+        const isHeightDefine = _.find(this.props.imageStyles, 'height');
+        const isWidthDefine = _.find(this.props.imageStyles, 'width');
```

Comment:
Think this unnecessary work, we have `this.props.size` passing it would be. the reset is handled by the below code
`const iconSize = StyleUtils.getAvatarSize(this.props.size);`

and avoid those unnecessary checks.
```js
 height={isHeightDefine ? isHeightDefine.height : iconSize}
                            width={isWidthDefine ? isWidthDefine.width : iconSize}
```

Reply:
I think this is necessary code, 
https://github.com/Expensify/App/blob/3faa6a77b076f6e6547340b93a2298042cf36266/src/components/AvatarWithImagePicker.js#L180-L184
have a look at this code
here we are using Avatar component and we are not passing size prop, so it will pass default size (is equal to 40)
const iconSize = StyleUtils.getAvatarSize(this.props.size);

so when image will fail to load ,then it will use 40 height and width at Icon, but that's wrong as here we are passing avatar size 80

Reply:
Sorry typo, we should have